### DISCUSSION TITLE
Make a .deb file

### DISCUSSION THREAD

**Discussion by brunwomelo** [grafana/grafana#48906] [General]
I was trying to update the UI/UX of the Grafana, to better fit the needs of the company I work with, and I found some useful tutorials as how to do that, and the documentation of this git helped me a lot, but in the end, to make the distribution of our new UI/UX easier, I was trying to create a .deb to install as something like 'myVersion.deb', that allowed me to install this version in our product. Can anyone help me with that, or have already tried something like that?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
